File: TFC-TourGuide-PHP/modelos/Guia.php
<?php

// Esta clase representa una Usuario

class Guia
{
    // Variables de clase
    private $id,$token ,$nombre, $apellidos,$ciudad,$provincia,$email,$puntuacion;

    // Constructor
    public function __construct($token,$nombre, $apellidos,$ciudad,$provincia,$email,$puntuacion)
    {
        $this->token = $token;
        $this->nombre = $nombre;
        $this->apellidos = $apellidos;
        $this->ciudad = $ciudad;
        $this->provincia = $provincia;
        $this->email = $email;
        $this->puntuacion = $puntuacion;
    }

    public function getProvincia(){
        return $this->provincia;
    }

    public function getToken(){
        return $this->token;
    }

    public function getNombre(){
        return $this->nombre;
    }

    public function getApellidos(){
        return $this->apellidos;
    }
    
    public function getCiudad(){
        return $this->ciudad;
    }

    public function getEmail(){
        return $this->email;
    }

    public function getPuntuacion(){
        return $this->puntuacion;
    }

    // Muestra los datos de la Usuario
    public function toString()
    {
        return
            [
                "id" => utf8_encode($this->id),
                "nombre" => utf8_encode($this->nombre),
                "apellidos" => utf8_encode($this->getApellidos),
                "ciudad" => utf8_encode($this->ciudad),
                "email" => utf8_encode($this->email)
            ];
    }
}

File: TFC-TourGuide-Android/app/src/main/java/com/example/tourguide/modelo/Guia.java
package com.example.tourguide.modelo;

public class Guia {
    private String id,nombre, apellido,email, ciudad,token,provincia,foto;
    private Double puntuacion;

    public Guia(String id,String token,String nombre, String apellido, String email, String ciudad,String provincia,Double puntuacion,String foto) {
        this.id = id;
        this.token = token;
        this.nombre = nombre;
        this.apellido = apellido;
        this.email = email;
        this.ciudad = ciudad;
        this.provincia = provincia;
        this.puntuacion = puntuacion;
        this.foto = foto;
    }

    public String getFoto() {
        return foto;
    }

    public void setFoto(String foto) {
        this.foto = foto;
    }

    public String getProvincia() {
        return provincia;
    }

    public void setProvincia(String provincia) {
        this.provincia = provincia;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getCiudad() {
        return ciudad;
    }

    public void setCiudad(String ciudad) {
        this.ciudad = ciudad;
    }

    public Double getPuntuacion() {
        return puntuacion;
    }

    public void setPuntuacion(Double puntuacion) {
        this.puntuacion = puntuacion;
    }

    @Override
    public String toString() {
        return "Guia{" +
                "id='" + id + '\'' +
                ", nombre='" + nombre + '\'' +
                ", apellido='" + apellido + '\'' +
                ", email='" + email + '\'' +
                ", ciudad='" + ciudad + '\'' +
                ", token='" + token + '\'' +
                ", provincia='" + provincia + '\'' +
                ", foto='" + foto + '\'' +
                ", puntuacion=" + puntuacion +
                '}';
    }
}
